stages/org.osbuild.os-release.image_id

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/python3
+"""
+Adds IMAGE_ID variable into /etc/os-release according to specification:
+    https://www.freedesktop.org/software/systemd/man/os-release.html#IMAGE_ID=
+"""
+
+import os
+import sys
+
+import osbuild.api
+
+SCHEMA = r"""
+"additionalProperties": false,
+"required": ["image_id"],
+"properties": {
+  "image_id": {
```

Comment:
Please define the `pattern` for this property based on the specification, which says `no spaces or other characters outside of 0–9, a–z, ".", "_" and "-"`